src/Tools/dotnet-monitor/ParameterCapturing/CapturedParameters.cs

```diff
@@ -28,7 +30,7 @@ public void AddParameter(ParameterInfo parameter)
             _parameters.Add(parameter);
         }
 
-        public string ActivityId { get; }
+        public string? ActivityId { get; }
 
         public ActivityIdFormat ActivityIdFormat { get; }
```

Comment:
Shouldn't this also be nullable now?

Reply:
The `ActivityIdFormat` enum has an `Unknown` value. From practice I noticed that's set when `ActivityId` is `null`